pkgs/applications/misc/mupdf/default.nix

```diff
@@ -24,7 +24,14 @@ in stdenv.mkDerivation rec {
 
   patches =
     # Use shared libraries to decrease size
-       stdenv.lib.optional (!stdenv.isDarwin) ./mupdf-1.14-shared_libs.patch
+    (stdenv.lib.optional (!stdenv.isDarwin) ./mupdf-1.14-shared_libs.patch)
+    ++
+    [( fetchpatch {
+        name = "CVE-2018-18662";
+        url = "http://git.ghostscript.com/?p=mupdf.git;a=patch;h=164ddc22ee0d5b63a81d5148f44c37dd132a9356";
```

Comment:
Is this url stable? It looks interesting :smile:

Reply:
yes it seems to be instable, we have something similar in other parts of nixpkgs. i think it is more clean than having the patchfile in the directory